components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Controller/DmApiPublisherMethodControllerTests.cs

```diff
@@ -221,5 +222,207 @@ public class DmApiPublisherControllerTests : TempFileProviderBase {
                     .BeTrue();
             }
         }
+
+        [Theory]
+        [InlineData("Controller/DmApiPayloadTwoEndpoints.json")]
+        public async Task DmApiGetConfiguredNodesOnEndpointAsyncTest(string publishedNodesFile) {
+            var endpointUrl = "opc.tcp://opcplc:50010";
+            var methodsController = await publishNodeAsync(publishedNodesFile);
+            var endpointRequest = new PublishNodesRequestApiModel();
+            endpointRequest.EndpointUrl = endpointUrl;
+            var response = await FluentActions
+                    .Invoking(async () => await methodsController
+                    .GetConfiguredNodesOnEndpointAsync(endpointRequest).ConfigureAwait(false))
+                    .Should()
+                    .NotThrowAsync()
+                    .ConfigureAwait(false);
+
+            Assert.True(response.Subject.Count == 2);
```

Comment:
>Assert [](http://example.com/codeflow?start=12&length=6)

Let's stay consistent and use FluentAssertions overall in this file.

Reply:
done